README.md

```diff
@@ -1,18 +1,49 @@
 
 ## Available Services
 
-### Document Upload Service
-- The [VBA document upload service](services/vba_documents) provides a 
-mechanism to submit VBA forms and documents to VA.
+### Address Validation*
+- The [Address Validation](https://developer.va.gov/explore/verification/docs/address_validation) service (for VA internal use only) provides methods to both standardize and validate addresses.
+
+### Appeals Status*
+- Use the [Appeals Status](https://developer.va.gov/explore/benefits/docs/appeals) API to request the status of a Veteran's benefits claim appeal. Currently for VA internal use only.
+
+### Authorization (OpenID Connect)
+- For [Health](https://developer.va.gov/explore/health/docs/authorization) and [Veteran Verification](https://developer.va.gov/explore/verification/docs/authorization).  The VA API Platform uses the OpenID Connect standard to allow Veterans to authorize third-party applications to access data on their behalf. 
+
+### Benefits Claims
+- Use the [Claims](https://developer.va.gov/explore/benefits/docs/claims) (Auto-Establishment) API to submit a Veteran's benefits claim via EVSS (Electronic Veterans Self-Service), and to request the status of a Veteran's benefits claim.
+
+### Benefits Intake (fka Document Upload Service)
+- The [Benefits Intake](https://developer.va.gov/explore/benefits/docs/benefits) API allows authorized third-party systems used by Veteran Service Organizations and agencies to upload claim documents (scanned to PDF) directly to the Veterans Benefits Administration's (VBA) claims intake process.
+
+### Disability Rating
+- The [Disability Rating](https://developer.va.gov/explore/verification/docs/disability_rating) API allows third-parties to access the disability rating of a Veteran after receiving authorization to do so using an Open ID Connect flow.
+
+
+### Facilities
+- Use the [Facilities](https://developer.va.gov/explore/facilities/docs/facilities) API to find contact info, location, hours of operation, available services and other relevant information about a specific VA facility. 
+
+### Health (currently dev only)
+- Use the [Health APIs](https://developer.va.gov/explore/health/docs/argonaut) to allow Veterans to view their medical records, schedule an appointment, find a specialty facility, and share their information with caregivers and providers.
+
+### Service History
+- The [Service History](https://developer.va.gov/explore/verification/docs/service_history) API allows third-parties to access the service history of a Veteran after receiving authorization to do so using an Open ID Connect flow.
+
+### Veteran Confirmation
+- The [Veteran Confirmation](https://developer.va.gov/explore/verification/docs/veteran_confirmation) API allows third-parties to request confirmation from the VA of an individual's Veteran status after receiving authorization to do so using an Open ID Connect flow.
+
+\* APIs marked with an asterisk are currently for internal VA use only.
```

Comment:
We should move this to the top of the  `Available Services` listings section. 

On that note, we should also remove wherever it says (for VA internal use only) since we have the asterisk.